teams/vsp/teams/insights-analytics/ga-events-data-dictionaries.md

```diff
@@ -742,116 +754,196 @@ _When the orientation of the way search results are presented is changed, either
-50|Profile Section Saved - Direct Deposit||User|true
-51|Health Facility Location||Hit|true
-52|Health Facility Type||Hit|true
-53|Global - Preferred Contact Method||Hit|true
-54|Global - Benefits Used Recently||Hit|true
-55|Education - Enrolled in Undergrad STEM||Hit|true
-56|Education - Pursue Teaching Certification||Hit|true
-57|Error Key||Hit|true
-58|Profile Section Saved - SMS Optin and Optout||User|true
-59|Placeholder - Event Number||Hit|false
-60|Product - Disability Ratings Viewed||User|true
-61|Video Title||Hit|true
-62|Product - Health Care VAOS User||User|true
-63|Product Area Used - Disability||User|true
-64|Education - Benefit Update||Hit|true
-65|Education - Benefit Claim for Self||Hit|true
-66|Education - National Call to Service||Hit|true
-67|Education - Applicant is Vet Tec||Hit|true
-68|Education - Sponsor Transferred Benefits||Hit|true
-69|Education - Receiving Sponsor Benefits||Hit|true
-70|Education - Sponsor Available||Hit|true
-71|Health Care VAOS - Appointment Type||Hit|true
-72|Health Care VAOS - Type of Care||Hit|true
-73|Health Care VAOS - Flow||Hit|true
-74|Education - 22-1995 - Exhausted All Benefits||Hit|true
-75|Health Care VAOS - Facility Type||Hit|true
-76|Health Care VAOS - Reason for Appointment||Hit|false
-77|Frequently Asked Question||Hit|true
-78|FAQ Section||Hit|true
+| Custom Dimension Name | Index | Scope | Definition |
+| --- | --- | --- | --- |
+| Internal User | 1 | User |  |
+| Product Area Used - Health Care | 2 | User |  |
+| Product Area Used - Benefits | 3 | User | Distinguish users who have ever completed at least one transaction in the Benefits product area |
+| Product Area Used - Education | 4 | User | Distinguish users who have ever completed at least one transaction in the Education product area |
+| Product Area Used - Other | 5 | User | Distinguish users who have ever completed at least one transaction in the Other product area |
+| Product Used - Education Forms | 6 | User | - |
+| Product Used - Health Care Application | 7 | User | - |
+| Product Used - Claims/Appeals Tracker | 8 | User | - |
+| Product Used - Secure Messaging | 9 | User | - |
+| Product Used - Rx Refills | 10 | User | - |
+| Product Used - VA Letters | 11 | User | - |
+| Product Used - Post 9/11 GI Bill Status | 12 | User | - |
+| Product Used - Facility Locator | 13 | User | - |
+| Product Used - GIBCT | 14 | User | - |
+| Product Used - Health Records | 15 | User | - |
+| Product Used - Burial Forms | 16 | User | - |
+| Product Used - Pension Forms | 17 | User | - |
+| Product Used - VIC | 18 | User | - |
+| Product Used - Pre-need Burial Application | 19 | User | - |
+| Product Used - Discharge Upgrade | 20 | User | - |
+| Logged In | 21 | Session | Distinguish users who have logged in |
+| LOA Level | 22 | Session | Level of authorization - collected right after a user logs in  |
+| Telephone Contact | 23 | User | User's phone contact information collected after a user logs in |
```

Comment:
@laurensimpson14 This gave me a bit of a scare at first 😅 , this is not the user's log in...this is the telephone number of when a phone number is clicked on